src/library/scala/runtime/LambdaDeserializer.scala

```diff
@@ -107,14 +105,10 @@ object LambdaDeserializer {
     factory.invokeWithArguments(captures: _*)
   }
 
-  private val ScalaSerializable = "scala.Serializable"
-
-  private val JavaIOSerializable = {
-    // We could actually omit this marker interface as LambdaMetaFactory will add it if
-    // the FLAG_SERIALIZABLE is set and of the provided markers extend it. But the code
-    // is cleaner if we uniformly add a single marker, so I'm leaving it in place.
-    "java.io.Serializable"
-  }
+  // We could actually omit this marker interface as LambdaMetaFactory will add it if
+  // the FLAG_SERIALIZABLE is set.
+  // But the code is cleaner if we uniformly add a single marker, so I'm leaving it in place.
+  private val Serializable = "java.io.Serializable"
```

Comment:
@smarter Thanks! I simplified things, could you have a look whether I should go further and get rid of this completely? As I understand, there now no benefit anymore of doing it, as the only marker we add is already added automatically.